### PR TITLE
Query update

### PR DESCRIPTION
**Pull request by chimwemwe007**
# Vet clinic database: query and update animals table
###  I have completed the following in this milestone

- Add a column species of type string to your animals table. Modify your schema.sql file.
- Add new queries to perform CRUD functions such as:
- 
1. Update the species column
2. Deleting date from the initial table
3. Remove data from existing table

## MAKE SPECIES 'unspecified'
unsoecified specie
unspecified species

## RECORD D
query-update
ELETION!

## ADD CORRECT SPECIES DATA
correct specie
ADD SPECIES SPECIES

## QUERY RESPONSES- part 1
queryresponse part1
QUERY RESPONSES

## QUERY RESPONSES-part 2
queryresponse part2

## RECORD DELETION!
recorddetete
RECORD DELETION

## RECORDS RECOVERY AFTER DELETION
RECOVERAFTER DELETE
RECORDS RECOVERY

## DELETE ANIMAL BORN AFTER JAN 2022
delete after jan
DELETE ANIMAL BORN after JAN 2022

## MULTIPLY WEIGHT BY -1
multiply by -1
MULTIPLIED BY -1

## ROLLBACK TO SAVEPOINT
rollback to savepoint
ROLLBACK TO SAVEPOINT

## UPDATE WEIGHT AFTER ROLLBACK
updateafterrollback
UPDATE WEIGHT AFTER ROLLBACK

## QUERY RESPONSES-1
queryresponse part1
QUERY RESPONSES

QUERY RESPONSES-2
queryresponse part2
QUERY RESPONSES-2

## RECORD DELETION
recordsrecovery

## RECORDS RECOVERY AFTER DELETION
RECOVERAFTER DELETE
RECORDS RECOVERY

## DELETE ANIMAL BORN AFTER JAN 2022
delete after jan
DELETE ANIMAL BORN IN 2022

## MULTIPLY WEIGHT BY -1
multiply by -1
WEIGHT MULTIPLIED BY -1

## ROLLBACK TO SAVEPOINT
rollback to savepoint
ROLLBACK TO SAVEPOINT

## UPDATE WEIGHT AFTER ROLLBACK
updateafterrollback
UPDATE WEIGHT AFTER ROLLBACK

## QUERY RESPONSES- part 1
queryresponse part1
QUERY RESPONSES

## QUERY RESPONSES-part 2
queryresponse part2
QUERY RESPONSES-part 2

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 CREATE TABLE animals (
   id INT NOT NULL PRIMARY KEY,
-  name VARCHAR(100) NOT NULL,
+  name VARCHAR(50) NOT NULL,
   date_of_birth DATE NOT NULL,
   escape_attempts INT NOT NULL,
   neutered BOOLEAN NOT NULL,
-  weight_kg NUMERIC(4, 2) NOT NULL
+  weight_kg NUMERIC(5, 2) NOT NULL,
+  species VARCHAR(50)
```
Comment (mikethreels) [inline schema.sql]: - [x] you included the species column in the original `CREATE TABLE` however you are expected to update the existing table by doing `ALTER TABLE animals ADD species varchar(255);`

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Chimwemwe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- professional looking readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (chimwemwe007) [reply schema.sql]: Hello Mike, I have added the species column to the animals table.

**Review by elmar8287 — APPROVED** [review]
Hi,  @chimwemwe007.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- All suggestions are iplemented

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**